cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -214,6 +215,49 @@ def test_embedded_ansible_repository_invalid_url_crud(request, appliance, wait_f
     repository.delete_if_exists()
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_add_private_repository_crud(
+    request, credentials_collection, appliance, wait_for_ansible
+):
+    """
+    Add Private Repository by using SCM credentials.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    creds = {
+        "username": conf.credentials["scm_creds"]["username"],
+        "password": conf.credentials["scm_creds"]["username"],
```

Comment:
Copy paste mistake I think , password should not be username

Reply:
Yes true :wink: